lib/time.dl

```diff
@@ -0,0 +1,15 @@
+extern type time
+
+extern function try_from_hms(hours: bit<8>, minutes: bit<8>, seconds: bit<8>): Result<time, string>
```

Comment:
I would prefix all these functions with `time_`. Alternatively, users can import this module using qualified name `import time as time`, but then what happens when you start adding support for dates and date-times?

Reply:
I've done everything but add this prefix.